library/std/src/sys/unix/cmath.rs

```diff
@@ -1,32 +1,31 @@
 #![cfg(not(test))]
 
-use libc::{c_double, c_float};
```

Comment:
These functions are used to implement methods of `f32` and `f64`, so anything else then `c_double = f64` and `c_float = f32` would be wrong anyway. This makes the file reusable on platforms that don't have `libc` available.